osbenchmark/resources/custom-operations.json.j2

```diff
@@ -0,0 +1,16 @@
+    {
+      "name": "index-append",
+      "operation-type": "bulk",
+      "bulk-size": {{bulk_size}},
+      "ingest-percentage": {{ingest_percentage}}
```

Comment:
We should still include the same Jinja defaults as the `default-operations.json.j2` file.